whole-tale/girder_wholetale
# During AinWT prepopulate Tale fields based on Dataverse metadata

Create nicer looking (metadata-wise) tales when importing from Dataverse.

### How to test
1. Import random AJPS supplement from dataverse, e.g. https://girder.local.wholetale.org/api/v1/integration/dataverse?siteUrl=https%3A%2F%2Fdataverse.harvard.edu%2F&datasetPid=doi%3A10.7910%2FDVN%2F26721&fullDataset=true&force=false
2. Verify that metadata (title, author, description) matches the original dataset's metadata
